packages/pwa-buildpack/lib/Utilities/graphQL.js

```diff
@@ -9,22 +11,35 @@ const fetchQuery = query => {
     const targetURL = new URL('graphql', process.env.MAGENTO_BACKEND_URL);
     const headers = {
         'Content-Type': 'application/json',
-        'Accept-Encoding': 'gzip'
+        'Accept-Encoding': 'gzip',
+        Accept: 'application/json',
+        'User-Agent': 'pwa-buildpack',
+        Host: targetURL.host
     };
 
     if (process.env.STORE_VIEW_CODE) {
         headers['store'] = process.env.STORE_VIEW_CODE;
     }
 
+    debug('Fetching query: %s', query);
```

Comment:
i think let's remove the debug statements now, eh? and just keep the headers.

Reply:
this one?

Reply:
These debug statements only run if the user whats them to. They are not console logs for instance. We usually have these debug lines in buildpack files to debug them when needed. For instance in this case I want to debug this file ill run the file with `DEBUG="*pwa-buildpack/lib/Utilities/graphQL*"`.